apps/nova/src/components/TutorProfile/Rating.tsx

```diff
@@ -4,21 +4,24 @@ import {
   ButtonType,
   Card,
   asFullAssetUrl,
-  useFormatMessage,
 } from "@litespace/luna";
 import { IRating } from "@litespace/types";
 import React from "react";
 import dayjs from "@/lib/dayjs";
 import { Edit, Star, Trash2 } from "react-feather";
 import { useRender } from "@/hooks/render";
 import EditRating from "./EditRating";
+import { profileSelectors } from "@/redux/user/profile";
+import DeleteRating from "./DeleteRating";
+import { useAppSelector } from "@/redux/store";
 
 const Rating: React.FC<{ rating: IRating.Populated }> = ({ rating }) => {
+  const profile = useAppSelector(profileSelectors.user);
   const edit = useRender();
   const deleteRating = useRender();
```

Comment:
For consistency, we can rename `edit` to `editRating`
```suggestion
  const editRating = useRender();
  const deleteRating = useRender();
```